Sources/MarkdownKit/Models/Nodes/Link.swift

```diff
@@ -11,18 +11,33 @@ public class Link: Node {
             return nil
         }
 
+        if isAutolink {
+            return childRange
+        }
+
         return NSRange(location: NSMaxRange(childRange) + 2, length: (url.absoluteString as NSString).length)
     }
 
     public var title: String? {
         cmark_node_get_title(node).map(String.init)
     }
 
+    public var isAutolink: Bool {
```

Comment:
Does this need to be public? I'm not sure what the use-case would be for consumers of this, given that `MarkdownKit.Link` abstracts over both `[]()` style links and these autolinks.

Reply:
Nope, I guess `internal` makes more sense here, what do you think `internal` or `private`?

Reply:
I think `internal` is fine if we care about the test case that uses it differentiating between autolinks and explicit links.  My gut says that since the two tests are named `testLinkDelimiters` and `testAutomaticLinks`, and the strings in those tests are explicitly different, that making this private should be fine.

Reply:
I made it public in a later commit 🤷

Reply:
(The rationale was maybe you'd want to style it differently than regular links so expose the data and let people decide.)